nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -292,3 +292,5 @@ In addition to numerous new and upgraded packages, this release has the followin
 - The option `services.prometheus.exporters.pihole.interval` does not exist anymore and has been removed.
 
 - `k3s` can now be configured with an EnvironmentFile for its systemd service, allowing secrets to be provided without ending up in the Nix Store.
+
+- The `i18n.inputMethod.fcitx` option has been removed. Please use `i18n.inputMethod.fcitx5` instead. `pkgs.fcitx` has also been removed.
```

Comment:
Please see the comment under the headline.

```suggestion
- The `i18n.inputMethod.fcitx` option has been replaced with `i18n.inputMethod.fcitx5` because fcitx 4 `pkgs.fcitx` has been removed.
```